_test_unstructured_client/test__decorators.py

```diff
@@ -1,70 +1,98 @@
-import os
 import pytest
 
 from unstructured_client import UnstructuredClient
+from unstructured_client.models import shared
+from unstructured_client.models.errors import SDKError
 
 
-def get_api_key():
-    api_key = os.getenv("UNS_API_KEY")
-    if api_key is None:
-        raise ValueError("""UNS_API_KEY environment variable not set. 
-Set it in your current shell session with `export UNS_API_KEY=<api_key>`""")
-    return api_key
+FAKE_KEY = "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"
 
 
 @pytest.mark.parametrize(
     ("server_url"),
     [
-        ("https://unstructured-000mock.api.unstructuredapp.io"), # correct url
+        ("https://unstructured-000mock.api.unstructuredapp.io"),  # correct url
         ("unstructured-000mock.api.unstructuredapp.io"),
         ("http://unstructured-000mock.api.unstructuredapp.io/general/v0/general"),
         ("https://unstructured-000mock.api.unstructuredapp.io/general/v0/general"),
         ("unstructured-000mock.api.unstructuredapp.io/general/v0/general"),
-    ]
+    ],
 )
 def test_clean_server_url_on_paid_api_url(server_url: str):
```

Comment:
I recommend you make test titles state the behavior being tested. In this case, something like:

```python
def test_it_replaces_a_misshapen_server_url_with_the_correct_url(...)
```
The name should be a verb-phrase, not a noun phrase.

I was able to work out roughly what this was doing by reading the code, but that's unnecessary work/time and is error prone. It's good to be able to read just the test names and get a good idea of behaviors the unit-under-test has. That can also help with identifying missing tests. A unit should do this and do that. It does not be this and be that.

A good title also helps a reader (importantly including yourself) to reason about whether all the necessary cases are being tested.

I know you didn't name this test in this PR but if you touch it you own it and you should feel free to remedy any shortcomings left by the last developer.